src/components/z-file-upload/index.tsx

```diff
@@ -98,14 +131,64 @@ export class ZFileUpload {
     ];
   }
 
-  render() {
+  renderUploadLink() {
+    return [
+      <input
+        onChange={() => this.fileInputHandler()}
+        type="file"
+        id="fileElem"
+        multiple
+        accept={this.acceptedFormat}
+        ref={(val) => (this.input = val)}
+      />,
+      <z-body variant="regular" level={1}>
+        Trascinalo qui o{" "}
+        <z-body
+          class="upload-link"
+          onClick={() => this.input.click()}
+          variant="semibold"
+          level={1}
+        >
+          caricalo
+        </z-body>{" "}
+        dal tuo computer
+      </z-body>,
+    ];
+  }
+
+  renderDefaultMode() {
     return (
-      <div class="container">
+      <div class={`container ${this.type}`}>
         {this.renderTitle()}
-        {this.renderDescriptionAndFileInfo()}
+        {this.renderDescription("semibold", 3)}
+        {this.renderAllowedFileExtensions()}
         {this.renderFileSection()}
-        {this.renderInput()}
+        {this.renderUploadButton()}
       </div>
     );
   }
+
+  renderDragDropMode() {
+    return (
+      <div class={`container ${this.type}`}>
```

Comment:
Perché il container e il title non li sposti nel render, visto che ci sono in entrambi i casi?